### ISSUE TITLE
Paginator select missing label

### ISSUE THREAD

**Issue by adplusverb** [thesmythgroup/vue-toolkit#26]
## Describe of the Issue

Paginator select element doesn't have a label. Add `aria-label="Items Per Page"`
